code/modules/spells/spell_types/shapeshift.dm

```diff
@@ -27,7 +27,10 @@
 			for(var/path in possible_shapes)
 				var/mob/living/simple_animal/A = path
 				animal_list[initial(A.name)] = path
-			shapeshift_type = input(M, "Choose Your Animal Form!", "It's Morphing Time!", null) as anything in animal_list
+			var/new_shapeshift_type = input(M, "Choose Your Animal Form!", "It's Morphing Time!", null) as null|anything in animal_list
+			if(shapeshift_type)
+				return
```

Comment:
`shapeshift_type` starts as null, so this check will always fail in the first time, but will prevent people that had multiple inputs open to select a new animal type